eduroam.md

```diff
@@ -40,98 +53,190 @@ The process of connecting to the network may be different depending on the Linux
 ```
 ctrl_interface=/var/run/wpa_supplicant
 ctrl_interface_group=wheel
 update_config=1
+
 network={
     ssid="eduroam"
     scan_ssid=1
     key_mgmt=WPA-EAP
+    proto=WPA2
+    pairwise=CCMP
+    group=CCMP
     eap=TLS
+    anonymous_identity="anonymous@rit.edu"
+    domain_match="radius.rit.edu"
     identity="abc1234@rit.edu"
     ca_cert="/location/of/RIT/cert"
-    client_cert="location/of/decryptedCert.pem"
-    private_key="location/of/decryptedKey.pem"
+    private_key="location/of/p12PrivateKey.p12"
+    private_key_passwd="P@ssw0rd1"
 }
 ```
 
-4. You're all set! Run the following commands in a terminal to start `wpa_supplicant`, then start `dhcpcd`. After this, your network should be set up properly. 
-```bash
-sudo wpa_supplicant -i [interface name] -c /etc/wpa_supplicant/[config file name]
-sudo dhcpcd [interface name]
+4. You're all set! Run the following commands in a terminal to start `wpa_supplicant`, then start `dhcpcd`. After this, your network should be set up properly.
+```
+# wpa_supplicant -B -i [interface name] -c /etc/wpa_supplicant/[config file name]
+# dhcpcd [interface name]
 ```
-Note that while this config file is persistent across reboots, as this guide stands, you will need to start `wpa_supplicant` and `dhcpcd` manually on each boot. For more information on starting `wpa_supplicant` at boot, and auto-starting `dhcpcd`, consult the [ArchWiki](https://wiki.archlinux.org/title/Wpa_supplicant#At_boot_(systemd)), or your distribution's wiki for non-`systemd` systems.
 
+For debugging purposes, the `-B` flag shown above can be omitted, allowing you to see the complete output of the command. Note that this omitting this flag will make the current terminal unusable, so a second session or terminal will be required to run the `dhcpcd` command. Also note that closing the terminal containing the `wpa_supplicant` command will also end the connection to the Wi-Fi, so act with care.
 
-## Alternate - RIT-Legacy
 
-If you are unable to connect using certificates, you can follow the [ITS RESNet manual registration instructions](https://www.rit.edu/its/resnet/manual-registration).
+Note that while this config file is persistent across reboots, as this guide stands, you will need to start `wpa_supplicant` and `dhcpcd` manually on each boot. For more information on starting `wpa_supplicant` at boot, and auto-starting `dhcpcd`, consult the [ArchWiki](https://wiki.archlinux.org/title/Wpa_supplicant#At_boot_(systemd)), or your distribution's wiki for non-`systemd` systems.
 
-## Quirks
+## Alternate - RIT-WiFi
 
-Based on the experiences of some RITlug members, it has been observed that RIT is using EAP-TLS authentication rather than MSCHAPV2, a practice which appears to be getting more common among other eduroam member universities.
```

Comment:
was there a particular reason for removing this? i know it was probably a bit salty and not particularly helpful, but mostly just curious

Reply:
It was a single sentence, in a section all-its-own, and the page already felt long enough.